### ISSUE TITLE
lowercase all String fields?

### ISSUE THREAD

**Issue by stakach** [PlaceOS/rubber-soul#2]
I've noticed that queries are case sensitive.
The analyzer we use in the ruby version

```
"analyzer": {
                "searchable": {
                    "type": "custom",
                    "tokenizer": "whitespace",
                    "filter": ["lowercase", "custom_ascii_folding"]
                },
```

Full file here: https://github.com/acaprojects/ruby-engine/blob/master/db/es_template.json
This is probably not the best implementation or even a valid one on the latest version of ElasticSearch - but might give us some ideas on improving the tokenisation process.

**Comment by caspiano** [comment]
Fixed by #6

**Comment by caspiano** [comment]
Fixed by #6
